modules/integration_aws-backup/conf/01-aws-backup-job-expired.yaml

```diff
@@ -0,0 +1,22 @@
+module: AWS Backup
```

Comment:
most of the comments from the previous detector apply here too.

Reply:
Addressed: https://github.com/claranet/terraform-signalfx-detectors/pull/457/files#diff-24c9e96f00206e3a5e0bd0a3f4aa55f47da745d831ca8f4a93ede35e8a4689f0R5-R17